src/core/hle/service/pctl/module.h

```diff
@@ -12,19 +13,33 @@ class System;
 
 namespace Service::PCTL {
 
+enum class Capability : s32 {
+    None = 0x0,
```

Comment:
```suggestion
    None = 0,
```

for consistency, given no other entries make use of hex literals.